astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
```

Comment:
I wouldn't laugh at you – I had to look it up! Is there a common format for listing wikipedia references? Maybe [like this](https://github.com/astropy/astropy/blob/6d0e5b050c5b71be9049ec662143f5b55b2a9a41/astropy/coordinates/baseframe.py#L1094)?